pyproject.toml

```diff
@@ -187,8 +182,6 @@ strict = true
 module = [
   "async_solipsism",
   "async_solipsism.*",
-  "grpc.aio",
-  "grpc.aio.*",
```

Comment:
The changes in this file  seem not related to `Avoid disabling some pylint checks when unnecessary`

Reply:
Moved to a new commit :+1: